cypress/utils/data_utils.ts

```diff
@@ -1,21 +1,29 @@
 import * as faker from "faker";
 
-export function getStakeholderName(): string {
+export function getFullName(): string {
     return faker.name.findName();
 }
 
-export function getStakeholderEmail(): string {
+export function getEmail(): string {
     return faker.internet.email();
 }
 
-export function getBusinessServiceName(): string {
+export function getCompanyName(): string {
     return faker.company.companyName();
 }
 
-export function getBusinessServiceDescription(): string {
+export function getSentence(): string {
     return faker.lorem.sentence();
 }
 
-export function getLongString(): string {
-    return faker.random.words(75);
+export function getJobTitle(): string {
+    return faker.name.jobType();
+}
+
+export function getRandomWord(length: number): string {
+    return faker.lorem.word(length);
+}
```

Comment:
as per my understanding, length argument should be renamed to charLength or something which will make more sense.
Because here we will get one word with given number of chars? right?
It confuses with method getRandomWords. Hence I think we need to update name or arguments accordingly.

Reply:
Its concrete - get me a random word of length n
These two methods exist as it is with same argument - length in faker library

Reply:
Updated